react/utils/metrics/useQuote.ts

```diff
@@ -0,0 +1,65 @@
+import type { Metric, SessionResponse } from './metrics'
+import { sendMetric } from './metrics'
+
+type UseQuoteFieldsMetric = {
+  quote_id: string
+  quote_reference_name: string
+  order_form_id: string
+  quote_creation_date: string
+  quote_use_date: string
+  creator_email: string
+  user_email: string
+  cost_center_name: string
+  buy_org_id: string
+  buy_org_name: string
+  quote_last_update: string
+}
+
+type UseQuoteMetric = Metric & { fields: UseQuoteFieldsMetric }
+
+export type UseQuoteMetricsParams = {
+  quoteState: Quote
+  orderFormId: string
+  account: string
+  sessionResponse: SessionResponse
+}
+
+const buildUseQuoteMetric = async (
```

Comment:
There's no async operation happening in this function, right? It can be a regular function.

Reply:
Right. I removed the fetching code and forgot to remove it.